backends-clickhouse/src/main/scala/io/glutenproject/expression/CHExpressionTransformer.scala

```diff
@@ -335,6 +335,39 @@ case class CHTruncTimestampTransformer(
   }
 }
 
+case class CHStringTranslateTransformer(
+    substraitExprName: String,
+    srcExpr: ExpressionTransformer,
+    matchingExpr: ExpressionTransformer,
+    replaceExpr: ExpressionTransformer,
+    original: StringTranslate)
+  extends ExpressionTransformer {
+
+  override def doTransform(args: java.lang.Object): ExpressionNode = {
+    // In CH, translateUTF8 requires matchingExpr and replaceExpr argument have the same length
```

Comment:
@exmy, could you take a review? I am assuming CH still have this limitation. cc @zzcclp

Reply:
yes, CH translateUTF8 has this limitation.